subnets/config.go

```diff
@@ -24,7 +24,7 @@ type Config struct {
 	// AllowedNodes is the set of node IDs that are explicitly allowed to connect to this Subnet when
 	// ValidatorOnly is enabled.
 	AllowedNodes        set.Set[ids.NodeID] `json:"allowedNodes"        yaml:"allowedNodes"`
-	ConsensusParameters snowball.Parameters `json:"consensusParameters" yaml:"consensusParameters,omitempty"`
+	ConsensusParameters snowball.Parameters `json:"consensusParameters" yaml:"consensusParameters"`
```

Comment:
Looks like this is reverting https://github.com/ava-labs/avalanchego/pull/3874/files so we must not be depending on this anywhere previously